docs/sources/installation/debian.md

```diff
@@ -11,8 +11,7 @@ Docker is supported on the following versions of Debian:
 
 ## Debian Jessie 8.0 (64-bit)
 
-Debian 8 comes with a 3.14.0 Linux kernel, and a `docker.io` package which
-installs all its prerequisites from Debian's repository.
+Debian 8 comes with a 3.16.0 Linux kernel, the `docker.io` package can be found in the `jessie-backports` repository. Reasoning behind this can be found <a href="https://lists.debian.org/debian-release/2015/03/msg00685.html" target="_blank">here</a>. Instructions how to enable the backports repository can be found <a href="http://backports.debian.org/Instructions/" target="_blank">here</a>.
```

Comment:
Ideally, the captions of the links should be something other than just "here". No huge issue, and it can be improved in a follow up PR.